CogNative/main.py

```diff
@@ -87,16 +86,31 @@
             exit(1)
     
     text = st.transcribe_audio(str(audio_path), dest_lang='english')
-
 else:
     # CHOOSE TEXT
     if '-dialogueText' in args and args.index('-dialogueText') < len(args):
         text = args[args.index('-dialogueText')+1]
     else:
         text = input("Enter text for voice clone:\n")
-        
-    tr = translation()
-    text = tr.translate_to(text, 'english')
+
+# TRANSLATE TEXT    
+tr = translation()
+curr_lang = tr.current_language(text)
+if curr_lang != 'en': # only translate text if it is not coming in as english
+    if len(text) >= 5000: # text file too large (>= 5000 characters), split api requests
+        text_split = split_text(text)
+        text_split_reduced = [""]
+        j = 0
+        for i, sentence in enumerate(text_split):
+            if len(text_split_reduced[j] + sentence) < 5000:
+                text_split_reduced[j] += sentence
+            else:
+                j += 1
+                text_split_reduced.append("")
```

Comment:
```suggestion
        curr_chunk = 0
        for sentence in text_split:
            if len(text_split_reduced[curr_chunk] + sentence) < 5000:
                text_split_reduced[curr_chunk] += sentence
            else:
                curr_chunk += 1
                text_split_reduced.append("")
```

Last change request. No need to enumerate if ```i``` isn't used. I just changed ```j``` to be more descriptive for anyone looking at this in the future.